### ISSUE TITLE
[vizuk] add published date

### ISSUE THREAD

**Issue by ngosang** [Jackett/Jackett#6832]
Published date is available in the search results, but in Spanish format.

Eg:
* 28-12-2019 13: 45
* day-month-year hour: min

**Comment by RicardoVelaC** [comment]
You are right but I think the problem here is only with the last torrent e.g.

Subido Hoy en 01: 43
Subido Ayer a las 10: 36
Subido Ayer a las 09: 18
Subido Friday a 23: 48
Subido Miércoles a 19: 13
Subido 29-12-2019 12: 51  ##For this format Jackett works fine but in the other cases Jackett show "0 min ago"

It's also happen in hechede.me

**Comment by ngosang** [comment]
> Subido 29-12-2019 12: 51 ##For this format Jackett works fine but in the other cases Jackett show "0 min ago"

It's not working for me. Try searching "joker".

**Comment by RicardoVelaC** [comment]
> It's not working for me. Try searching "joker".

You're right it's don't work for Joker:

Joker [2019][BDRip 1080p][Dual AC3_5.1][Sub Forz]
Subido 27-12-2019 19: 02

I don't know how to fix it

**Comment by RicardoVelaC** [comment]
> Joker [2019][BDRip 1080p][Dual AC3_5.1][Sub Forz]
> Subido 27-12-2019 19: 02

Maybe you can fix it in vizuk.yml changing args for date

args: "02-01-2006 15:04"
args: "02-01-2006 15: 04"

**Comment by garfield69** [comment]
fixed the date processing and can process English too.

**Comment by garfield69** [comment]
Jackett 0.12.1470

**Comment by ngosang** [comment]
@garfield69 Only the dates with `02-04-2019 20: 26` format are working. I tested in Spanish and English with the same result.
1. Tested with `empty search` and `joker`. I get many results with 0m.
2. If you use the same expression you are using for English, you don't need 2 cases => https://github.com/Jackett/Jackett/blob/b891b24e26bcfb66c7f4a0096faba71cc577b46d/src/Jackett.Common/Definitions/vizuk.yml#L176
```
selector: td.torrent_name:contains("Subido"):not(:contains("-")), .torrentOwner:contains("Subido"):not(:contains("-"))
```

**Comment by garfield69** [comment]
Huh. 
(1)
if I search for joker I get 4 results from 10 days ago
![image](https://user-images.githubusercontent.com/6933240/71802573-ec08a680-30c2-11ea-8f73-f7edba04c6cf.png)
and an empty search gets me 25 results with just one 0m
![image](https://user-images.githubusercontent.com/6933240/71802606-0e022900-30c3-11ea-955a-3ff08cf9ba14.png)
and that is probably because of my timezone.

nothing more I can do about it. I think its is better than before. ;-b

(2)
I do not understand what you are referring to.
the dual comma delimited selectors are because we are supporting two style themes, and the selectors are different.
the language support comes from looking for **Subido** or **Uploaded**.

**Comment by ngosang** [comment]
@garfield69 
(1) This PR fixes all remaining issues => #6845 Could you take a look at hachede? @RicardoVelaC says something similar is happening. Maybe this PR solves it too.
(2) I meant that => #6846

**Comment by garfield69** [comment]
I don't have  an a/c on hachede but given that vizuk was modelled from the hachede indexer I will apply the same set of changes and trust that it will work there too.

**Comment by garfield69** [comment]
Jackett 0.12.1486

**Comment by RicardoVelaC** [comment]
If you set "Modern" option in view torrent list on Profile preferences in the tracker config, you get 0 results with false date "0 min ago" in "Classic view" I always get someone "0 min ago"

**Comment by garfield69** [comment]
this should be fixed once 95654695594202e17b65deed248cf8aa11541a22 is published

**Comment by ngosang** [comment]
Jackett pre-release 0.12.1525
